webpack.test.config.js

```diff
@@ -1,6 +1,14 @@
-module.exports = require('./.config/webpack/base.js')(
+const config = require('./.config/webpack/base.js')(
     {
         definitions: ['TEST_COPY_PASTE']
     },
     'development'
-);
\ No newline at end of file
+);
+
+config.module.rules.forEach(rule => {
+    if (rule.loader) {
+        rule.loader = rule.loader.replace('ts-loader', `ts-loader?${JSON.stringify({ transpileOnly: true })}`);
+    }
```

Comment:
This is a bit hackish but it prevents TS from building up a types mapping, which is very costly in terms of memory usage.. while it's good to do so in the actual build, doing so in Cypress provides very little value and just causes a `heap out of memory` exception after a while.